lib/theme/app_constants.dart

```diff
@@ -1,5 +1,16 @@
 class Metrics {
   static const defaultBorderRadius = 12.0;
   static const defaultHorizontalPadding = 24.0;
+  static const defaultHeight = 16.0;
+  static const defaultPadding = 20.0;
   static const formFieldHeight = 56.0;
+  static const avatarSize = 36.0;
+  static const primaryButtonSize = 56.0;
+
+  static const double spacing4 = 4;
+  static const double spacing16 = 16;
+  static const double spacing20 = 20;
+  static const double spacing28 = 28;
+
+  static const homePageIndicatorSize = 8.0;
```

Comment:
IMO, the `Metrics` should only contain constants that will be used across screens and widgets. They become a part of the design system besides fonts, colors, etc.; if we tend to place _every_ width, height, spacing, etc., here, it becomes messy quickly 🙏 

In this case, consider the following:
- put `avatarSize` to `home_header`.
- put `homePageIndicatorSize` to `home_page_indicator`.
- remove `primaryButtonSize` and use `formFieldHeight` instead.
- remove `defaultHeight` and use `spacing16` instead in `home_footer`.

Also, we can let Dart infer the type instead explicitly declaring, for example, `static const spacing4 = 4.0;` This is merely a convention, though, so it can be discussed to decide by the team 🙏

Reply:
Good suggestion 🙏.
However, for this point, `remove primaryButtonSize and use formFieldHeight instead.`, do you think it may be confusing when the size of the button gets from the size of a Form Field? I mean the size should be got from some value related to buttons like `primaryButtonSize` or `secondaryButtonSize` instead of a Form field 😄 . 
For now, I will move the button size variable to the local of `home_footer_widget` 🙏 
Also, I updated all the comments at https://github.com/nimblehq/flutter-ic-khanh-thieu/pull/56/commits/d7d50d4fd276c0bc9206b2710cd9892a038fdb8e